.bin/bashbrew-buildkit-env-setup.sh

```diff
@@ -67,7 +67,8 @@ _bashbrew_buildkit_env_setup() {
 			# https://hub.docker.com/r/docker/scout-sbom-indexer/tags
 			sbomTag="$(grep <<<"$externalPins" -m1 '^docker/scout-sbom-indexer:')"
 			sbomTag="$(_resolve_external_pins "$sbomTag")"
-			vars="$(_jq_setenv <<<"$vars" BASHBREW_BUILDKIT_SBOM_GENERATOR "$sbomTag")"
+			scanners="php-composer-lock,erlang-otp-application,lua-rock-cataloger,swipl-pack-cataloger,opam-cataloger"
+			vars="$(_jq_setenv <<<"$vars" BASHBREW_BUILDKIT_SBOM_GENERATOR "$sbomTag,\"EXTRA_SCANNERS=$scanners\"")"
```

Comment:
Repeating a few minor suggestions, this time with more explanation for each one (so they're hopefully easier to understand/discuss): :innocent: :heart:

- renaming `sbomTag` to `sbomGenerator`, since that's the variable it's setting so it's a more appropriate name

- using Bash's `+=` to append the scanners directly to that variable to avoid a second variable _and_ so we can use single-quotes so that the CSV quoting is easier to get right (no escaping necessary)

- adding a comment pointing to https://github.com/moby/buildkit/pull/5372 so we can easily find where this functionality was added to BuildKit + a note about what "EXTRA_SCANNERS" is (because it's not documented publicly anywhere)

```bash
			local sbomGenerator
			# https://hub.docker.com/r/docker/scout-sbom-indexer/tags
			sbomGenerator="$(grep <<<"$externalPins" -m1 '^docker/scout-sbom-indexer:')"
			sbomGenerator="$(_resolve_external_pins "$sbomGenerator")"
			# https://github.com/moby/buildkit/pull/5372 - "EXTRA_SCANNERS" is an optional parameter to the Scout SBOM Indexer
			sbomGenerator+=',"EXTRA_SCANNERS=php-composer-lock,erlang-otp-application,lua-rock-cataloger,swipl-pack-cataloger,opam-cataloger"'
			vars="$(_jq_setenv <<<"$vars" BASHBREW_BUILDKIT_SBOM_GENERATOR "$sbomGenerator")"
```

I also think we should probably wait to merge this until we have a new release of `docker/scout-sbom-indexer` that includes support for `EXTRA_SCANNERS` so we don't forget that this isn't actually doing anything yet and scratch our heads later. :thinking:  Perhaps even include the swap from the `1-doi` tag to the `1` tag?